src/constants/gameplay.asm

```diff
@@ -122,6 +131,11 @@ DIALOG_GOT_GUARDIAN_ACORN equ $05
 ; Cooldown time for dialog boxes
 DIALOG_COOLDOWN              equ $18
 
+; Flag for denoting a dialog as unskippable,
+; set as the high bit of the dialog bank
+; (see dialog_banks.asm)
+DIALOG_UNSKIPPABLE equ $80
+
```

Comment:
Nitpick: maybe all those additions point to the need of a new `constant/dialog.asm` file? (It could be in a later PR though)